wiki/Tournaments/OMIC/2019_4K/id.md

```diff
@@ -0,0 +1,320 @@
+---
+tags:
+- O!MIC 4K 2019
+- OMIC
+---
+# osu!mania 4K Indonesia Cup 2019
+![O!MIC 4K 2019 Logo](img/logo.png)
+**osu!mania 4K Indonesia Cup 2019** adalah turnamen osu!mania 4K tunggal khusus wilayah Indonesia dengan memakai sistem pertandingan *Royal Rumble* pada babak setelah Kualifikasi, lalu satu lawan satu dengan *Double-Elimination bracket* dimulai dari Penyisihan Grup.
+## Jadwal Turnamen
```

Comment:
```suggestion

## Jadwal Turnamen

```
make sure all of the headers are spaced like this (an extra newline before and after). I think that's what fajar13k was suggesting